maestro-app/src/main/resources/file_centric.json

```diff
@@ -79,25 +79,52 @@
           "file_autocomplete"
         ]
       },
+      "file_type": {
+        "type": "keyword"
+      },
+      "file_access": {
+        "type": "keyword"
+      },
       "analysis" : {
         "properties" : {
-          "id" : {
+          "analysis_id" : {
             "type": "keyword",
             "copy_to": [
               "file_autocomplete"
             ]
           },
+          "analysis_type": {
+            "type": "keyword"
+          },
+          "analysis_version": {
+            "type": "integer"
+          },
+          "workflow": {
+            "properties": {
+              "workflow_name": {
+                "type": "keyword"
+              },
+              "workflow_version": {
+                "type": "keyword"
+              }
+            }
+          },
           "experiment": {
-            "type": "object"
+            "type": "object",
+            "properties": {
+              "platform": {
+                "type": "keyword"
+              },
+              "library_strategy": {
+                "type": "keyword"
+              }
+            }
```

Comment:
this should be removed, as it's part of dynamic schema

Reply:
https://github.com/icgc-argo/argo-metadata-schemas/blob/master/mappings/maestro_file_centric.json#L96